docs/AccessControl.rst

```diff
@@ -153,8 +153,8 @@ A tuple type's access level is the minimum of the access levels of its
 elements. A function type's access level is the minimum of the access levels of
 its input and return types.
 
-A typealias may have any access level up to the access level of the type it
-aliases. That is, a ``private`` typealias can refer to an ``public`` type, but
+A type-alias may have any access level up to the access level of the type it
```

Comment:
Isn't `typealias` one word already a Swift keyword? I think this makes sense to leave as-is.